lib/index.js

```diff
@@ -41,12 +42,23 @@ Server.prototype._createServer = function() {
     return;
   }
 
+  // connections
+  self._connections = {};
+
   var server = net.createServer(function(connection) { //'connection' listener
     new Connection(connection, self, {
       version: self.protocolVersion,
       authKey: self.authKey,
       protocol: self.protocol
     });
+    // assign id
+    connection.id = self.nextConnectionId++;
```

Comment:
as I mentioned - then there's no way to get that ID later in callback (L60), unless I'm missing something